Sql/1v1_Battlemaster.sql

```diff
@@ -1,2 +1,3 @@
-INSERT INTO `creature_template` (`entry`, `difficulty_entry_1`, `difficulty_entry_2`, `difficulty_entry_3`, `KillCredit1`, `KillCredit2`, `modelid1`, `modelid2`, `modelid3`, `modelid4`, `name`, `subname`, `IconName`, `gossip_menu_id`, `minlevel`, `maxlevel`, `exp`, `faction`, `npcflag`, `speed_walk`, `speed_run`, `scale`, `rank`, `mindmg`, `maxdmg`, `dmgschool`, `attackpower`, `DamageModifier`, `BaseAttackTime`, `RangeAttackTime`, `unit_class`, `unit_flags`, `unit_flags2`, `dynamicflags`, `family`, `trainer_type`, `trainer_spell`, `trainer_class`, `trainer_race`, `minrangedmg`, `maxrangedmg`, `rangedattackpower`, `type`, `type_flags`, `lootid`, `pickpocketloot`, `skinloot`, `resistance1`, `resistance2`, `resistance3`, `resistance4`, `resistance5`, `resistance6`, `spell1`, `spell2`, `spell3`, `spell4`, `spell5`, `spell6`, `spell7`, `spell8`, `PetSpellDataId`, `VehicleId`, `mingold`, `maxgold`, `AIName`, `MovementType`, `InhabitType`, `HoverHeight`, `HealthModifier`, `ManaModifier`, `ArmorModifier`, `RacialLeader`, `movementId`, `RegenHealth`, `mechanic_immune_mask`, `flags_extra`, `ScriptName`, `VerifiedBuild`) VALUES (999990, 0, 0, 0, 0, 0, 23766, 0, 0, 0, 'Arena Battlemaster 1v1', '', '', 8218, 70, 70, 2, 35, 1048577, 1.1, 1.14286, 1, 0, 252, 357, 0, 304, 1, 2000, 2000, 1, 768, 2048, 0, 0, 0, 0, 0, 0, 215, 320, 44, 7, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, '', 0, 1, 1, 1, 1, 1, 0, 0, 1, 0, 0, 'arena1v1_worldscript', 12340);
+DELETE FROM `creature_template` WHERE `entry` IN (999991);

```

Comment:
??? @milestorme why?
I don't see why but you can maybe explain it.

Reply:
Ok according to xdev and my testing this npc does nothing , and can be removed only 1 npc is needed for this script , hence not needed

Reply:
What if someone has created a creature_template ID with the same ID, here it just deletes the row but doesn't add anything on top of it, very confusing

Reply:
I think it's unsafe @milestorme

Reply:
Your referencing my original typo I delete 99991 and the sql insert adds 99991 not 99990 that was an early typo I fixed later as we had 2 entries before check my latest pr1 I changed that half way through

Reply:
You want I can remove the delete function just incase someone has used this custom id for something else

Reply:
I just removed it for safety

Reply:
In fact, my first comment was erroneous because (I'm ill and) I didn't check properly the initial SQL. But after a 2 sec analysis, I think it's still better to not delete, because if someone has used that ID, then it will print an error and the user will just need to change it and that's fine considering the SQL file is only 2 lines long.

What do you think?

Reply:
Yeah I agree , I removed it